mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/HeaderCollection.java

```diff
@@ -81,7 +82,7 @@ private Builder() {
 
         public Builder add(Field field) {
             Preconditions.checkNotNull(field);
-            String headerName = field.getName().toLowerCase();
+            String headerName = field.getName().toLowerCase(Locale.US);
```

Comment:
Nope. Internationalization is not supported anywhere else. It's not part of this task.

This task is "get a predictable behaviour on toLowerCase". It's very different from "i8n everywhere"